tests/IntegrationTests/Controllers/ProductsUIControllerViewIntegrationTests.cs

```diff
@@ -0,0 +1,89 @@
+﻿using DataAccessLayer;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc.Testing;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using static IntegrationTests.Helpers.WebPageHelpers;
+
+namespace IntegrationTests.Controllers
+{
+    [TestFixture]
+    public class ProductsUIControllerViewIntegrationTests
+    {
+        private readonly string _requestUri;
+        private readonly HttpClient _httpClient;
+        private readonly WebApplicationFactory<WebUI.Startup> _factory;
+
+        public ProductsUIControllerViewIntegrationTests()
+        {
+            _requestUri = "/Products";
+            _factory = new WebApplicationFactory<WebUI.Startup>()
+                .WithWebHostBuilder(builder => 
+                builder.ConfigureServices(services =>
```

Comment:
Please fix indentation by shifting this statement on one tab (line 28-34)

Reply:
done